pkg/virt-handler/vm.go

```diff
@@ -526,8 +526,14 @@ func (d *VirtualMachineController) updateVMIStatus(vmi *v1.VirtualMachineInstanc
 					ifc := v1.VirtualMachineInstanceNetworkInterface{
 						Name: network.Name,
 						IP:   podIface.PodIP,
-						IPs:  []string{podIface.PodIP},
+						IPs:  []string{},
 					}
+					for _, ip := range podIface.PodIPs {
+						if ip != "" {
```

Comment:
Is this ever a case? It should list IPs only.

Reply:
it is a case where there is just one ip, but i am working on the change that was purposed in one of the comments here, so it will be removed once all done.